runtime/unix.c

```diff
@@ -49,6 +49,10 @@
 #ifdef __APPLE__
 #include <mach-o/dyld.h>
 #endif
+#include <pthread.h>
+#if defined(DEBUG) || defined(NATIVE_CODE)
+#include <execinfo.h>
+#endif
```

Comment:
`execinfo.h` is very specific to Glibc.  This must be protected by a `#ifdef HAS_EXECINFO_H` and a configure-time test.  Or it should be removed.

Reply:
I'll remove the `caml_print_trace` function. I'd introduced it when the DWARF backtraces didn't exist. They're no longer needed.

Reply:
Removed in https://github.com/ocaml/ocaml/pull/10831/commits/6aa1379ba5a4891e4eb76ad6baafcb02ecd616a8.